nft_ingester/src/processors/account_based/mpl_core_processor.rs

```diff
@@ -133,7 +134,14 @@ impl MplCoreProcessor {
             })
             .unwrap_or((0, &default_creators));
 
-        let mut plugins_json = serde_json::to_value(&asset.plugins)
+        // convert HashMap plugins into BTreeMap to have always same plugins order
+        // for example without ordering 2 assets with same plugins can have different order saved in DB
+        // it affects only API response and tests
+        let ordered_plugins: BTreeMap<_, _> = asset.plugins
```

Comment:
wdyt of sorting when you're building the response? The json and order are a bit tricky, as you're implicitly relying on the reading code not to decode it as a map again.